tools/requirements_mypy.txt

```diff
@@ -1,5 +1,6 @@
 mypy==1.14.0
 mypy-extensions==1.0.0
+pydantic==2.10.6
```

Comment:
This should be in requirements_metadata.txt, not requirements_mypy.txt, right?

Reply:
We also need it for running mypy, otherwise we're missing the type definitions.